### PR TITLE
[bitnami/postgresql-ha] Fix readiness-probe script

### PR DESCRIPTION
**Pull request by DevHwan**
### Description of the change

This PR fixes incorrect `readiness-probe.sh` script which prevents the pod from being in ready state.

### Benefits

Properly initialize postgres pods with following option
```yaml
postgresql:
  headlessWithNotReadyAddresses: true
```

### Possible drawbacks

None

### Applicable issues

N/A

### Additional information

When deployed with `headlessWithNotReadyAddresses: true` enabled, following kubernetes events will occur.
```
Events:
  Type     Reason     Age                   From               Message
  ----     ------     ----                  ----               -------
  Normal   Scheduled  18m                   default-scheduler  Successfully assigned oscmp/pg-postgresql-ha-postgresql-0 to gaia-dev-control-plane
  Normal   Pulling    18m                   kubelet            Pulling image "docker.io/bitnami/postgresql-repmgr:15.4.0-debian-11-r31"
  Normal   Pulled     18m                   kubelet            Successfully pulled image "docker.io/bitnami/postgresql-repmgr:15.4.0-debian-11-r31" in 34.720127739s
  Normal   Created    18m                   kubelet            Created container postgresql
  Normal   Started    18m                   kubelet            Started container postgresql
  Warning  Unhealthy  18m                   kubelet            Readiness probe failed: /readiness-probe.sh: line 23: command failed: command not found
  Warning  Unhealthy  3m43s (x97 over 18m)  kubelet            Readiness probe failed: /readiness-probe.sh: line 46: return: can only `return' from a function or sourced script
```

There are two errors in `readiness-probe.sh` script file.
 1. `postgresql_execute ` command is not valid in `readiness-probe.sh` scope.
 2. `exit` command should be used in bash script file.

This PR changes the script return statement to exit function and db status check command.

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/postgresql-ha/templates/postgresql/hooks-scripts-configmap.yaml [hunk 1]
```diff
@@ -109,8 +109,7 @@ data:
     TMP_FIRST_READY_FILE_TS="/tmp/ts-first-ready.mark"
     TMP_DELAY_APPLIED_FILE="/tmp/delay-applied.mark"
 
-
-    DB_CHECK_RESULT=$(echo "SELECT 1" | postgresql_execute "$POSTGRESQL_DATABASE" "$POSTGRESQL_USERNAME" "$POSTGRESQL_PASSWORD" "-h 127.0.0.1 -tA" || "command failed")
```
Comment (andresbono) [inline bitnami/postgresql-ha/templates/postgresql/hooks-scripts-configmap.yaml:113]: Any idea why postgresql_execute is not available? It should be loaded from `. /opt/bitnami/scripts/libpostgresql.sh`...

**Review by andresbono — COMMENTED** [review]
Thank you for your contribution! I just left a question, just in case you know.

File: bitnami/postgresql-ha/templates/postgresql/hooks-scripts-configmap.yaml (hunk 1 above)
Comment (DevHwan) [reply bitnami/postgresql-ha/templates/postgresql/hooks-scripts-configmap.yaml:113]: Ok let me check for it.

Comment (andresbono) [reply bitnami/postgresql-ha/templates/postgresql/hooks-scripts-configmap.yaml:113]: Hi @DevHwan, any updates on this? If not, please let us know and we will try to investigate it on our side.

Comment (DevHwan) [reply bitnami/postgresql-ha/templates/postgresql/hooks-scripts-configmap.yaml:113]: Sorry, I did forgot to submit the review..

**Review by andresbono — APPROVED** [review]
Thank you very much for your contribution. LGTM. I have rebase the branch and bumped the chart version again. The PR is ready to be merged. Thanks!

**Pull request merged**